rpyc/core/netref.py

```diff
@@ -148,6 +148,11 @@ def __getattribute__(self, name):
                 return self.__dir__()
             elif name in _deleted_netref_attrs:
                 raise AttributeError()
+            elif name == "__array__":  # Speedup array copy if the remote is an array
+                if hasattr(self, '__array__'):
```

Comment:
Note, this line causes an infinite recursion, since `hasattr` is based on `getattr` mechanism which causes `__getattribute__` to be called.